Auth0/ControllerModalPresenter.swift

```diff
@@ -24,18 +24,12 @@ import UIKit
 
 struct ControllerModalPresenter {
 
-    let rootViewController: UIViewController?
-
-    init(rootViewController: UIViewController? = UIApplication.shared.keyWindow?.rootViewController) {
-        self.rootViewController = rootViewController
-    }
-
     func present(_ controller: UIViewController) {
         topViewController?.present(controller, animated: true, completion: nil)
     }
 
     var topViewController: UIViewController? {
-        guard let root = rootViewController else { return nil }
+        guard let root = UIApplication.shared.keyWindow?.rootViewController else { return nil }
```

Comment:
I'd keep the old behaviour of allowing it to pass a root controller and if its nil, default to the one from `UIApplication` when there is the need to present a VC